src/models.py

```diff
@@ -36,19 +31,21 @@ def check_password(self, password_to_check: str) -> bool:
         """Use this method to check User's password."""
         return self._get_password_hash(password_to_check) == self.password_hash
 
-    def create_user(self, username: str, password: str, session: scoped_session) -> None:
+    @classmethod
+    def create_user(cls, username: str, password: str) -> None:
         """Use this method to create a new user."""
-        self.username = username
-        self.set_password(password)
-        session.add(self)
+        new_user = cls(username=username, password_hash=hashlib.sha256(password.encode()).hexdigest())
```

Comment:
Instead of
```python
password_hash=hashlib.sha256(password.encode()).hexdigest()
```
we can use our `_get_password_hash()` method here.

Reply:
This change isn't done here for some reason. Probably it's because of I missed this and merged PR without this change.

It will be done as a part of another ticket - #90